packages/pools/src/constants/boostedPools/index.ts

```diff
@@ -0,0 +1,22 @@
+import { ChainId } from '@pancakeswap/chains'
+
+import { arbBoostedPools } from './42161'
+
+import { BoostedPoolsSupportedChainId } from '../boostedPoolsSupportedChains'
+import { isBoostedPoolsSupported } from '../../utils/boosted/isBoostedPoolsSupported'
+import { BoosterConfig } from '../../utils/boosted/types'
+
+export type BoostedPoolsConfigByChain<TChainId extends ChainId> = {
+  [chainId in TChainId]: BoosterConfig[]
```

Comment:
we can simply add optional mark here if not all the chain has the config
```suggestion
  [chainId in TChainId]?: BoosterConfig[]
```